aws/logs_monitoring/README.md

```diff
@@ -365,6 +365,10 @@ To test different patterns against your logs, turn on [debug logs](#troubleshoot
 
 `AdditionalTargetLambdaARNs`
 : Comma separated list of Lambda ARNs that will get called asynchronously with the same `event` the Datadog Forwarder receives.
+`InstallFromLayer`
+: Disable to use legacy installation flow, (via Github), for regions where Lambda layers aren't supported, such as regions in China. Defaults to true.
+`LayerARN`
+: ARN for the layer containing the forwarder code. If empty, the script will use the version of the layer the forwarder was published with. Defaults to empty.
 
```

Comment:
These docs make it sound like the `InstallFromLayer` option determines whether or not we install from the layer. But looking at the template, it seems that we automatically don't install from the layer when we detect the installation is happening in AWS China or the `SourceZipUrl` parameter is set.

Also, why do we assume that the customer wants to use a Zip if they set a `SourceZipUrl`, but we don't necessarily assume they want to use a layer if they set `LayerARN`?

Reply:
I could add a check the the China condition, to see if LayerARN was manually set. The reason for having the China condition there is so user's won't have to think decide which flow to use by default. I can probably remove the SourceZipUrl condition.